phpunit.xml.dist

```diff
@@ -19,6 +19,12 @@
         <!-- ###+ symfony/sendgrid-mailer ### -->
         <!-- MAILER_DSN=sendgrid://KEY@default -->
         <!-- ###- symfony/sendgrid-mailer ### -->
+
+        <!-- ###+ lexik/jwt-authentication-bundle ### -->
+        <env name="JWT_SECRET_KEY" value="%kernel.project_dir%/config/jwt/private.pem"/>
+        <env name="JWT_PUBLIC_KEY" value="%kernel.project_dir%/config/jwt/public.pem"/>
+        <env name="JWT_PASSPHRASE" value="ca03baf2622f91b4a9a050cc8296a898158ede7665e80109ce372e45b5bcc59e"/>
```

Comment:
kkkkkk For that problem, I opened a issue to regenerate that key soon as possible. For now, I don't have any problem with that because is just develop moment of that application.